solutions/swb-reference/src/services/sshKeyService.test.ts

```diff
@@ -53,8 +53,8 @@ describe('SshKeyService', () => {
     let listUserSshKeysForProjectRequest: ListUserSshKeysForProjectRequest;
     beforeEach(() => {
       listUserSshKeysForProjectRequest = {
-        projectId: 'proj-123',
-        userId: 'user-123'
+        projectId: 'proj-1234abcd-1111-abcd-1234-abcd1234abcd',
```

Comment:
I'd suggest you save mock values (mockProjectId, mockUserId) as constants at the top most describe()

Reply:
Good point! Would have saved me some finding these values!